### DISCUSSION TITLE
Concordance search

### DISCUSSION THREAD

**Discussion by shrihari-k** [WeblateOrg/weblate#10095] [Q&A]
Do we have a feature of concordance search available in Weblate?

**Comment by nijel** [comment]
Yes, you can perform a concordance search on the translation memory in the automatic suggestions tab, see https://docs.weblate.org/en/latest/user/translating.html#automatic-suggestions.

**Reply by shrihari-k** [reply]
Thank you! What ever I Search there I do not get any results. Any thing I am missing in configuration?

**Reply by nijel** [reply]
Perhaps https://docs.weblate.org/en/latest/admin/projects.html#contribute-to-shared-translation-memory?
